### PR TITLE
locksmithctl: support etcd.service and etcd2.service

### PR DESCRIPTION
**Pull request by philips**
In order to support the best-effort use case we need to now check for
both etcd.service and etcd2.service to figure out if the user has etcd
running.

Fixes https://github.com/coreos/locksmith/pull/73 for https://github.com/coreos/bugs/issues/317

### CODE REVIEW

**Comment by philips** [comment]
Manually tested on a three machine cluster with a fake etcd2.service that actually was just etcd.service copied across under various scenarios:

etcd not running:

```
Apr 08 00:25:13 core-01 locksmithd[491]: Unlocking old locks failed: [etcd.service etcd2.service] are inactive. Retrying in 10s.
```

etcd2 running:

```
Apr 08 00:29:56 core-02 locksmithd[646]: etcd2.service is active
Apr 08 00:29:56 core-02 locksmithd[646]: Logins detected, delaying reboot for 5 minutes.
Apr 08 00:29:58 core-02 systemd[1]: Stopping Cluster reboot manager...
Apr 08 00:30:41 core-02 locksmithd[507]: Unlocked existing lock for this machine
```

```
Apr 08 00:31:29 core-02 locksmithd[507]: etcd.service is active
Apr 08 00:31:30 core-02 locksmithd[507]: Logins detected, delaying reboot for 5 minutes.
Apr 08 00:32:15 core-02 locksmithd[494]: Unlocked existing lock for this machine
```

**Comment by philips** [comment]
cc @jonboulle @crawford

**Comment by marineam** [comment]
[LGTM](http://www.lgtm.in/i/0cwmbgERp)

**Comment by crawford** [comment]
LGTM

File: locksmithctl/daemon.go [hunk 1]
```diff
@@ -139,23 +146,33 @@ func setupLock() (lck *lock.Lock, err error) {
 }
 
 // etcdActive returns true if etcd is not in an inactive state according to systemd.
-func etcdActive() (running bool, err error) {
+func etcdActive() (active bool, name string, err error) {
```
Comment (jonboulle) [inline locksmithctl/daemon.go]: etcdNotInactive :-)

**Comment by jonboulle** [comment]
lgtm

**Pull request merged**